x/service/types/params.go

```diff
@@ -1,10 +1,16 @@
 package types
 
 import (
+	sdkerrors "cosmossdk.io/errors"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v2"
 )
 
+// DefaultAddServiceFee is the default value for the add service fee
+// parameter in the genesis state of the service module.
+// TODO: Revisit default param values
+const DefaultAddServiceFee = 1000000000 // 1000 POKT
```

Comment:
@Olshansk can you explain this name change to me? This would imply there is a variable fee to add a service - if there is a min there should also be a maximum no?